### ISSUE TITLE
Add an accessible name to the mobile login link

### ISSUE THREAD

**Issue by Ovgodd** [suitenumerique/docs#2318]
## Feature Request

**Is your feature request related to a problem or unsupported use case? Please describe.**

In the mobile header, the login link is displayed as an icon-only link.

The icon is visible, but the link does not have an accessible name.

RGAA Rule : 6.1 - Chaque [lien](https://accessibilite.numerique.gouv.fr/methode/glossaire/#lien) est-il explicite (hors cas particuliers) ?

Current markup:

```html
<a target="_blank" href="https://docs.numerique.gouv.fr/api/v1.0/authenticate/">
  <span aria-hidden="true"><svg>…</svg></span>
</a>
```

Because the icon is hidden from assistive technologies with `aria-hidden="true"`, screen reader users cannot understand the purpose of the link.

**Describe the solution you'd like**

Add an accessible name to the link using `aria-label`.

Expected markup:

```html
<a
  target="_blank"
  href="https://docs.numerique.gouv.fr/api/v1.0/authenticate/"
  aria-label="Se connecter à Docs - nouvelle fenêtre"
>
  <span aria-hidden="true"><svg>…</svg></span>
</a>
```

The accessible name should clearly describe the link destination and mention that it opens in a new window.

**Do you want to work on it through a Pull Request?**

Yes.

**Comment by Ovgodd** [comment]
This ticket will be handle in the landingpage repos : https://github.com/numerique-gouv/lasuite-landingpage/issues/197
